Rubberduck.Refactorings/EncapsulateField/EncapsulationStrategies/ConvertFieldsToUDTMembers.cs

```diff
@@ -27,14 +18,11 @@ public ConvertFieldsToUDTMembers(IDeclarationFinderProvider declarationFinderPro
             _stateUDTField = model.ObjectStateUDTField;
         }
 
-        protected override void ModifyFields(IEncapsulateFieldRewriteSession refactorRewriteSession)
+        protected override void ModifyFields(IRewriteSession refactorRewriteSession)
         {
             var rewriter = refactorRewriteSession.CheckOutModuleRewriter(_targetQMN);
 
-            foreach (var field in SelectedFields)
-            {
-                refactorRewriteSession.Remove(field.Declaration, rewriter);
-            }
+            rewriter.RemoveVariables(SelectedFields.Select(f => f.Declaration).Cast<VariableDeclaration>());
```

Comment:
I would add a line break for each LINQ call. However, that is a question of style.